pyproj/crs.py

```diff
@@ -62,13 +62,60 @@ def _dict2string(projparams):
             pjargs.append("+{key}".format(key=key))
         elif value is False:
             pass
-        # make sure string starts with proj or init
-        elif not proj_inserted and key in ("init", "proj"):
-            pjargs.insert(0, "+{key}={value}".format(key=key, value=value))
-            proj_inserted = True
         else:
             pjargs.append("+{key}={value}".format(key=key, value=value))
-    return " ".join(pjargs)
+    return _from_string(" ".join(pjargs))
+
+
+def _from_string(in_crs_string):
```

Comment:
Something like `_proprocess_string` might be a better descriptor as function name